save_discard_button/static/src/scss/indicator_button.scss

```diff
@@ -0,0 +1,64 @@
+.o_form_status_indicator_buttons {
+    margin-left: 10px;
+    .o_form_button_save,
+    .o_form_button_cancel {
+        width: 100%;
+        &.o_form_button_save {
+            background-color: $primary;
+            &:hover {
+                filter: brightness(0.9);
+            }
+        }
+        &.o_form_button_cancel {
+            margin-left: 5px;
+            &:hover {
+                background-color: #f6f7fa;
+                border-color: #e0e2e6;
+            }
+        }
+        .fa-cloud-upload {
+            width: 100%;
+            &:before {
+                content: "Save";
```

Comment:
Is this translatable?

Reply:
This is not translatable, because in css not allowed to translate.